jsapp/js/components/submissions/mediaCell.scss

```diff
@@ -1,6 +1,6 @@
  @import "scss/_colors";
 
- .media-cell {
+.media-cell {
   text-align: center;
```

Comment:
I think we need to switch this to `left` for the icon + text case. Also the icon button hover state background shape is not perfect, see:

<img width="131" alt="Screenshot 2021-08-18 at 23 08 45" src="https://user-images.githubusercontent.com/2521888/129978762-d332988d-5fce-4e28-b2f8-7b15dbceaaf0.png">

Reply:
I tried the `text-align: left` but I think this extra height is coming from the `line-height` of the text. I got around it by adding a wrapper `div` so that `div` eats the line height instead. Maybe this is a hacky way to do it though :P